mod/profiles/set/type/user.rb

```diff
@@ -1,6 +1,9 @@
 include_set Abstract::TwoColumnLayout
 include_set Abstract::Thumbnail
 
+CONTRIBUTION_TYPES = %i[metric_value metric wikirate_company project
+                        source wikirate_topic research_group].freeze
+
```

Comment:
@xithan, fyi researcher.rb makes heavy use of constants now, thanks to your comments. (that got merged in)